docs/pbc/all/price-management/202404.0/base-shop/extend-and-customize/configuration-of-price-modes-and-types.md

```diff
@@ -80,19 +80,55 @@ You can use these keys in environment configuration:
    		<input type="hidden" name="referrer-url" value="{% raw %}{{{% endraw %} app.request.requestUri {% raw %}}}{% endraw %}" />
    	</form>
    {% raw %}{%{% endraw %} endif {% raw %}%}{% endraw %}
-   ```
+```
+
+
+The switch can happen only if quote have to items.
+
+This is available after the product currency release so you must first follow the steps in [Upgrade the Price module](/docs/pbc/all/price-management/{{site.version}}/base-shop/install-and-upgrade/upgrade-modules/upgrade-the-price-module.html).
```

Comment:
Is this a prerequisite then?

Reply:
it's so old, so I'd even remove this completely. it was needed in 2021